bower.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "ls-js-sdk",
+  "version": "1.0.0",
+  "homepage": "https://github.com/Lifestreams/ls-js-sdk",
+  "authors": [
+    "Lukas Nowacki <lucjan.nowacki@gmail.com>"
+  ],
+  "description": "JavaScript HTTP Wrapper provides a set of methods to interact with Lifestreams API in a browser context.",
+  "main": "dist/api.min.js",
+  "moduleType": [
+    "globals"
+  ],
+  "keywords": [
+    "lifestreams",
+    "timeline",
```

Comment:
I think these are good. Once we submit, is there any ability to change the keywords?